src/main/scala/com/databricks/sparkdl/Models.scala

```diff
@@ -0,0 +1,179 @@
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+  private[sparkdl] object InceptionV3 extends NamedImageModel {
+    override val name = "InceptionV3"
+    override val height = 299
+    override val width = 299
+    override val graphInputNode = name + "_input"
+    override val graphOutputNode = name + "_sparkdl_output__"
+
+    override def graph: GraphDef = ModelFetcher.getFromWeb(
+        "https://s3-us-west-2.amazonaws.com/spark-deep-learning-models/sparkdl-InceptionV3_v1.pb",
+        fileName = "sparkdl-inceptionV3_v1.pb",
+        base64Hash = "8OImhbhTEoacQhEKRDjavT0xpMPH2pH3HQrOAZ1NUtA="
+    )
+  }
+
+  /**
+   * Model provided by Keras. All cotributions by Keras are provided subject to the
+   * MIT license located at https://github.com/fchollet/keras/blob/master/LICENSE
+   * and subject to the below additional copyrights and licenses.
+   *
+   * The MIT License (MIT)
+   *
+   * Copyright (c) 2016 Shaoqing Ren
+   *
+   * Permission is hereby granted, free of charge, to any person obtaining a copy
+   * of this software and associated documentation files (the "Software"), to deal
+   * in the Software without restriction, including without limitation the rights
+   * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   * copies of the Software, and to permit persons to whom the Software is
+   * furnished to do so, subject to the following conditions:
+   *
+   * The above copyright notice and this permission notice shall be included in all
+   * copies or substantial portions of the Software.
+   *
+   * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+   * SOFTWARE.
+   */
+  private[sparkdl] object ResNet50 extends NamedImageModel {
+    override val name = "ResNet50"
+    override val height = 224
+    override val width = 224
+    override val graphInputNode = name + "_input"
+    override val graphOutputNode = name + "_sparkdl_output__"
+
+    override def graph: GraphDef = ModelFetcher.getFromWeb(
+        "https://s3-us-west-2.amazonaws.com/spark-deep-learning-models/sparkdl-ResNet50_v1.pb",
+        fileName = "sparkdl-inceptionV3_v1.pb",
+        base64Hash = "yrnfcbi84GxJjo30eGkl27GAHxig0JsC106v5wQr3gE="
+    )
+  }
+
+  /**
+    * Model provided by Keras. All cotributions by Keras are provided subject to the
```

Comment:
indent

Reply:
About the import reordering (git does not let me responded above :/ )

Thanks, the Idea setup helps! I think Idea actually did the right thing in alphabetizing imports from third party libraries. The only violations of the rules is the scala.* reference at the bottom.